### PR TITLE
Added Auto Emoji Feature + implemented the resource pack inside the jar file

### PR DESCRIPTION
**Pull request by Sherko231**
Hi ! 
So I liked this plugin but wanted to make it kinda more practical 
so I added `emoji-phrases.yml` file 
whenever someone say in his message for example : "haha" .... the joy emoji will appear automatically 
of course everything is customizable from `emoji-phrases.yml`
this feature can be cancelled from `config.yml` file
finally .... i added the resource pack inside the jar file under assets/resource_pack .... so the users don't need to download the resource_pack independently 

hope you like it 😅

### CODE REVIEW

**Comment by smartcmd** [comment]
Wow that's cool!

**Review by smartcmd — APPROVED** [review]

**Comment by smartcmd** [comment]
Broken building...

**Comment by smartcmd** [comment]
image
This should be removed

File: pom.xml [hunk 1]
```diff
@@ -56,8 +56,7 @@
         <dependency>
             <groupId>cn.powernukkitx</groupId>
             <artifactId>powernukkitx</artifactId>
-            <version>5f01581497</version>
-            <scope>provided</scope>
+            <version>1.20.10-r1</version>
```
Comment (smartcmd) [inline pom.xml:59]: Why no `<scope>provided</scope>`

**Review by smartcmd — CHANGES_REQUESTED** [review]
image
Also remove this

File: pom.xml (hunk 1 above)
Comment (Sherko231) [reply pom.xml:59]: idk it didn't work on my ide

**Comment by Sherko231** [comment]
try now

**Pull request merged**